### DISCUSSION TITLE
Does the latest release work manually with Drop_target.bat?

### DISCUSSION THREAD

**Discussion by Kchaff** [apprenticeharper/DeDRM_tools#2050] [General]
I'm a bit confused (what's new, right?) .... have not updated DeDRM tools in years...and moving to a new computer now...and want to install the latest.

I do not use the Calibre Plugin but just remove DRM manually using the drop_target.bat (older version 6.5.5) on Windows 11 .... maybe the drop_target.bat doesn't even exist in the latest version???

I downloaded and it seem that it can only be used with Clibre plugin??

Thanks for any help

**Comment by ElleKayEm** [comment]
You're correct.  The stand-alone apps were dropped with DeDRM 6.7.  Only the calibre plugins are included in the later versions.  The latest version isn't actually at this repo anymore.  Current development is here:  https://github.com/noDRM/DeDRM_tools

**Reply by Kchaff** [reply]
Thanks for the verification! Much appreciated. Old version has been working fine so will have to evaluate and choose. :)

Thank you for the new link as well!
